### PR TITLE
[#7927] Moonrepo icon added

### LINKED ISSUE #7927: Moon repo icon
### Brand Name

Moon

### Website

https://moonrepo.dev

### Popularity Metric

GitHub stars are 1K,
https://www.similarweb.com/website/moonrepo.dev/#ranking

Moon is an up-and-coming monorepo tool which is an alternative to NX or turborepo

### Official Resources for Icon and Color

brand style guide: https://github.com/moonrepo/moon/blob/master/website/static/brand/style-guide.pdf

logo: https://github.com/moonrepo/moon/blob/master/website/static/brand/icon.svg
invert: https://github.com/moonrepo/moon/blob/master/website/static/brand/icon-vector.svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by wadehrarshpreet**
moonrepo-3

**Issue:** closes #7927

**Similarweb rank:** 6,036,501
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
HI @wadehrarshpreet,

perfect fit:
image

Left some notes on the `source` tag, if this is fixed its ready to merge!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7159,6 +7159,11 @@
             "hex": "00945E",
             "source": "https://www.moo.com/uk/about/press"
         },
+        {
+            "title": "Moonrepo",
+            "hex": "6F53F3",
+            "source": "https://raw.githubusercontent.com/moonrepo/moon/master/website/static/brand/icon-vector.svg"
```
Comment (dirien) [inline _data/simple-icons.json:7165]: Can you change this to: `https://moonrepo.dev`

**Review by dirien — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (wadehrarshpreet) [reply _data/simple-icons.json]: resolved @dirien

**Review by dirien — APPROVED** [review]
LGTM!

**Pull request merged**